python/lsst/ts/ess/csc/ess_csc.py

```diff
@@ -117,9 +118,18 @@ async def _read_loop(self) -> None:
                     sensor_data = data[common.Key.TELEMETRY]
                     await self.process_telemetry(data=sensor_data)
                 else:
-                    raise ValueError(f"Unknown data {data!r} received.")
-        except Exception:
+                    self.fault(
+                        code=ErrorCode.TelemetryError,
+                        report=f"Unknown data {data!r} received.",
```

Comment:
They are very uncommon because the sending code always uses the same enum for the values that it can send. So I replaced this with logging an error and continuing.